tests/paddle/test_operators.py

```diff
@@ -6,10 +6,10 @@
 import pytest
 import paddle
 
-from utils import assert_allclose, create_fp8_meta
-import transformer_engine_paddle as tex
 from transformer_engine.paddle.cpp_extensions import cast_to_fp8, cast_from_fp8, gemm, fp8_gemm
 from transformer_engine.paddle.fp8 import is_fp8_available
+from utils import assert_allclose, create_fp8_meta    # pylint: disable=wrong-import-order
+import transformer_engine_paddle as tex    # pylint: disable=wrong-import-order
```

Comment:
This is a similar situation to https://github.com/NVIDIA/TransformerEngine/issues/115 . Pylint suggest to move `import transformer_engine_paddle as tex` ahead of `transformer_engine.paddle`. However, that gives the error "libtransformer_engine.so not found in LD_LIBRARY_PATH".

Reply:
One solution would be https://github.com/NVIDIA/TransformerEngine/pull/249/commits/8ceef3c12c09e9ce9b93fa962c300d817d63b6be . Do you think it is necessary?

Reply:
I don't like exposing `transformer_engine_paddle` within `transformer_engine.paddle` since the extension should be treated as an implementation detail and should not be accessed by users. The corresponding TensorFlow test handles this by relying on the base TE module to load libtransformer_engine.so:

https://github.com/NVIDIA/TransformerEngine/blob/80825fde111cca493240f3217030ee22135a285a/tests/tensorflow/test_extensions.py#L11-L12

I suppose some ugliness might be inevitable here since this test is testing internal implementations.

Reply:
I made a new change here: https://github.com/NVIDIA/TransformerEngine/pull/249/commits/a11bd4aed65886e3951aeeff172e6ebd58dee295 . Unfortunately, I cannot find a way to get rid of `wrong-import-order` warning. According to my testing, pylint reports the same warning on this line: https://github.com/NVIDIA/TransformerEngine/blob/80825fde111cca493240f3217030ee22135a285a/tests/tensorflow/test_extensions.py#L12  I think [TransformerEngine/tests/tensorflow/test_extensions.py](https://github.com/NVIDIA/TransformerEngine/blob/80825fde111cca493240f3217030ee22135a285a/tests/tensorflow/test_extensions.py#L11-L12) did not pass pylint, because pylint reports many other problems on it, e.g., missing-function-docstring.